tests/testdata/codegen/simplified-invokes/nodejs/absMultiArgs.ts

```diff
@@ -29,5 +29,5 @@ export function absMultiArgsOutput(a: pulumi.Input<number>, b?: pulumi.Input<num
         "a": a,
         "b": b,
     };
-    return pulumi.output(args).apply((resolvedArgs: any) => absMultiArgs(resolvedArgs.a, resolvedArgs.b, opts))
+    return pulumi.output(args).apply((resolvedArgs: any) => absMultiArgs(resolvedArgs.a, resolvedArgs.b, opts));
```

Comment:
Can this fix be pulled into it's own little PR?

Reply:
Yep, will do